scalableminds/webknossos
# Make centerNewNode compatible with merger mode and also influence delete behavior

### URL of deployed dev instance (used for testing):
- https://fixcentersettinginmergermode.webknossos.xyz

### Steps to test:
- Enable "Center new Nodes" and delete/create nodes --> active node should be centered automatically
  - test the same in merger mode
- Disable "Center new Nodes" and delete/create nodes --> active node should be NT centered automatically
  - test the same in merger mode

### Issues:
- fixes #5398 

------
